toolbox/core/bst_memory.m

```diff
@@ -255,11 +255,12 @@
         SurfaceFile = sDbSurf.FileName;
     end
     % Get subject and surface type
-    [sSubject, iSubject, iSurfDb] = bst_get('SurfaceFile', SurfaceFile);
+    [sSubject, iSubject, iSurfDb] = db_get('SurfaceFile', SurfaceFile);
     if isempty(iSubject)
         SurfaceType = 'Other';
     else
-        SurfaceType = sSubject.Surface(iSurfDb).SurfaceType;
+        sAnatomy = db_get('AnatomyFile', iSurfDb);
+        SurfaceType = sAnatomy.SurfaceType;       

```

Comment:
No need to get all fields when you only use SurfaceType.

Reply:
Solved in 0e12a3c